src/seacat-auth/components/ControlledSwitch/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './switch.scss';
+
+// documentation available in asab-webui/doc/switch-components.md
+
+const Switch = ({ isOn, toggle, disabled=false, title, size='md' }) => {
```

Comment:
How is switch connected with seacat-auth?

Reply:
I dont think it should be under the seacat-auth section

Reply:
To me it looks like a misunderstanding in the separation of the components into appropriate parts

Reply:
This component is used in ControlledSwitchWithAuthz. I asked you about this and you said to move all components that are used by seacat-auth to this folder. Same is for `UncontrolledSwitch` this components is used in `UncontrolledSwitchWithAuthz`